MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Button.xaml

```diff
@@ -133,12 +133,27 @@
         <Setter Property="Foreground" Value="{DynamicResource PrimaryHueDarkForegroundBrush}"/>
     </Style>
 
-    <Style x:Key="MaterialDesignRaisedAccentButton" TargetType="{x:Type ButtonBase}" BasedOn="{StaticResource MaterialDesignRaisedButton}">
+    <Style x:Key="MaterialDesignRaisedSecondaryButton" TargetType="{x:Type ButtonBase}" BasedOn="{StaticResource MaterialDesignRaisedButton}">
         <Setter Property="Background" Value="{DynamicResource SecondaryHueMidBrush}"/>
         <Setter Property="BorderBrush" Value="{DynamicResource SecondaryHueMidBrush}"/>
         <Setter Property="Foreground" Value="{DynamicResource SecondaryHueMidForegroundBrush}"/>
     </Style>
 
+    <Style x:Key="MaterialDesignRaisedSecondaryLightButton" TargetType="{x:Type ButtonBase}" BasedOn="{StaticResource MaterialDesignRaisedButton}">
+        <Setter Property="Background" Value="{DynamicResource SecondaryHueLightBrush}"/>
+        <Setter Property="BorderBrush" Value="{DynamicResource SecondaryHueLightBrush}"/>
+        <Setter Property="Foreground" Value="{DynamicResource SecondaryHueLightForegroundBrush}"/>
+    </Style>
+
+    <Style x:Key="MaterialDesignRaisedSecondaryDarkButton" TargetType="{x:Type ButtonBase}" BasedOn="{StaticResource MaterialDesignRaisedButton}">
+        <Setter Property="Background" Value="{DynamicResource SecondaryHueDarkBrush}"/>
+        <Setter Property="BorderBrush" Value="{DynamicResource SecondaryHueDarkBrush}"/>
+        <Setter Property="Foreground" Value="{DynamicResource SecondaryHueDarkForegroundBrush}"/>
+    </Style>
+
+    <Style x:Key="MaterialDesignRaisedAccentButton" TargetType="{x:Type ButtonBase}" BasedOn="{StaticResource MaterialDesignRaisedSecondaryButton}">
```

Comment:
Thanks for keeping the old keys in there. I am going to flag these for removal in the 5.0.0 release.